### PR TITLE
Added Playlist management

### PR DESCRIPTION
**Pull request by jeetsdev**
## Added playlist management feature 

- User can create a playlist 
- User can go to the playlist page to see all the playlist 
- User can delete an existing playlist 
- User can add videos to the playlist
- User can delete those videos from the playlist 

[Preview]()

### CODE REVIEW

File: src/contexts/playlist-context.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+								authorization: authToken,
+							},
+						},
+					);
+
+					const {
+						status,
+						data: { playlist },
+					} = res;
+					console.log("res: ", res);
```
Comment (rahulmistry751) [inline src/contexts/playlist-context.js]: remove console log

**Review by rahulmistry751 — APPROVED** [review]
Every functionalities are working fine.

**Review by ayush65 — APPROVED** [review]
Code looks clean 
Functionality is working as expected

**Review by Mandhularajitha — APPROVED** [review]
remove console

**Pull request merged**